### ISSUE TITLE
Ruby 2.7 warning: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by sedubois** [twalpole/apparition#36]
```rb
~/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/apparition-0.4.0/lib/capybara/apparition/driver/launcher.rb:11: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

**Comment by menge101** [comment]
I have been having issues with this as well.

Also the related warning:

$HOME/.asdf/installs/ruby/2.7.0/lib/ruby/gems/2.7.0/gems/apparition-0.4.0/lib/capybara/apparition/page.rb:468: warning: The called method `call' is defined here

It is dumping so many useless warnings to my terminal that it becomes difficult to actually find the real output of my capybara suite.  Approximately 8 warnings per call into apparition.

apparition (0.4.0)
cucumber (3.1.2)
capybara (3.30.0)

**Comment by menge101** [comment]
The issue can be worked around by calling ruby with the -w0 flag.

RUBYOPT="-w0" rspec or RUBYOPT="-w0" cucumber respectivley.

ruby -w0 whatever.rb

**Comment by dentarg** [comment]
@menge101 or `-W:no-deprecated` or `Warning[:deprecated] = false` in the code (from https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/)

**Comment by twalpole** [comment]
Sorry for the delays here - Fixing the 2.7 warnings in Capybara took precedence and real life has got in the way of fixing here too.  Hopefully soon (or PRs if anyone else wants to try tackling it)

**Comment by menge101** [comment]
@twalpole I forked it for that purpose, but running the existing specs on my system (an admittedly fairly weak hp laptop) slammed my system and I had to kill the process.

**Comment by twalpole** [comment]
Fixed via 170a98529bb3d49ee1a4335b43c183ee2eb9de41 - use with Capybara master branch currently
